src/main/java/com/bitdecay/game/component/DebugCircleComponent.java

```diff
@@ -19,6 +20,18 @@ public DebugCircleComponent(MyGameObject obj, Color color, float radius){
         this.radius = radius;
     }
 
+    public DebugCircleComponent(MyGameObject obj, Config conf) {
+        super(obj, conf);
+        this.radius = new Double(conf.getDouble("radius")).floatValue();
+        Config colorConf = conf.getConfig("color");
+        this.color = new Color(
+                new Double(colorConf.getDouble("r")).floatValue(),
+                new Double(colorConf.getDouble("g")).floatValue(),
+                new Double(colorConf.getDouble("b")).floatValue(),
+                new Double(colorConf.getDouble("a")).floatValue()
```

Comment:
Do these have to have the `new Double()` wrapped around them? What does `conf.getDouble(...)` return?

(Not really worried about it other than it looked a little strange to me)